docs/examples/binding_lifetime/binding_lifetime.rst

```diff
@@ -0,0 +1,233 @@
+
+    profile_2 = double_exponential_fit.profile_likelihood()
+
+    plt.figure()
+    profile_2.plot()
+    plt.tight_layout()
+    plt.show()
+
+.. image:: profile1.png
+
+The profile likelihood for the single exponent is almost symmetric, while for the double exponential it is very skewed.
+According to the likelihood profile, a single exponential fit is better.
+
+Bootstrapping
+^^^^^^^^^^^^^
+
+Bootstrapping is typically used in combination with maximum likelihood fitting to determine the uncertainty intervals for the fitted parameter and further test the quality of the fit.
+
+Compute and plot the bootstrapping the distribution for the single exponential fit::
+
+    bootstrap1 = exponential_fit.calculate_bootstrap(iterations=1000)
+
+    plt.figure()
+    bootstrap1.hist(alpha=0.05)
+
+.. image:: bootstrap1.png
+
+The fitted binding lifetime is 4 seconds has a 95% confidence interval of (3,5.2) seconds.
+
+Compute and plot the bootstrapping the distribution for the double exponential fit::
+
+    bootstrap2 = double_exponential_fit.calculate_bootstrap(iterations=1000)
+
+    plt.figure()
+    bootstrap2.hist(alpha=0.05)
+
+.. image:: bootstrap2.png
+
+The bootstrapping distribution for double exponential fit is sometimes bimodal. Furthermore, the component :math:`a_2` has a peak around zero. 
+This further indicates that a single exponential function best fits the data.
+
+Bayesian Information Criterion
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Typically adding more parameters (components) to a model, will make the fit better. However, having too many parameters can lead to overfitting. 
+The Bayesian information Criterion (BIC) quantifies the quality of the fit by looking at the value of the likelihood function and penalizes the addition of parameters.
+
+The BIC for the single and double exponential fit are respectively given by::
+
+    >>> print(exponential_fit.bic)
+    >>> print(double_exponential_fit.bic)
+
+    1317.4882114486545
+    1309.1566330421654
+
+The BIC value for the double exponential fit is minimal, but the difference is smaller than 10, so the evidence is not super strong.
+
+Conclusion
+----------
+
+We fitted a single exponential and double exponential. We used the likelihood profile, bootstrapping and BIC to determine the optimal model.
+The likelihood profile and bootstrapping distributions indicated that a single exponential is the optimal model, while the BIC indicated that a double exponential is the optimal model.
+Looking at Figure with the :ref:`double_exponential_fit <double exponential fit>`, there are only a few data points larger than 20 seconds that support the second component. 
+
+Therefore we conclude that the optimal model is a single exponential with :math:`\tau = 4` seconds with a 95% confidence interval of (3,5.2) seconds.
+
+Note that the optimal model can change as more data is recorded; for fitting two more exponentials without overfitting, typically more data points are needed.
```

Comment:
I like that you mention that these results change as more data becomes available.